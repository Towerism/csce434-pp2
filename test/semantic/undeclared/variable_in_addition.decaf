int asdf() {
    0 + a;
}