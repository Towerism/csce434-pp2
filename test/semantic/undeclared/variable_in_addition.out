
*** Error line 2.
    0 + a;
        ^
*** No declaration found for variable 'a'

